module/solutions/helper.js

```diff
@@ -291,9 +300,14 @@ module.exports = class SolutionsHelper {
 	 * @returns {JSON} solution creation data.
 	 */
 
-	static createSolution(userToken, solutionData, checkDate = false) {
+	static createSolution(solutionData, checkDate = false) {
```

Comment:
@Prajwal17Tunerlabs check where else we this function is used , because param order change can affect those uses. Please verify once

Reply:
we are not using createSolution elsewhere, so this change will not harm.